d2cli/main.go

```diff
@@ -738,6 +756,76 @@ func renderPDF(ctx context.Context, ms *xmain.State, plugin d2plugin.Plugin, opt
 	return svg, nil
 }
 
+func renderPPTX(ctx context.Context, ms *xmain.State, presentation *pptx.Presentation, plugin d2plugin.Plugin, opts d2svg.RenderOpts, outputPath string, page playwright.Page, diagram *d2target.Diagram, boardPath []string) error {
+	var currBoardPath []string
+	// Root board doesn't have a name, so we use the output filename
+	if diagram.Name == "" {
+		currBoardPath = append(boardPath, getFileName(outputPath))
+	} else {
+		currBoardPath = append(boardPath, diagram.Name)
+	}
+
+	if !diagram.IsFolderOnly {
+		// gofpdf will print the png img with a slight filter
+		// make the bg fill within the png transparent so that the pdf bg fill is the only bg color present
+		diagram.Root.Fill = "transparent"
+
+		svg, err := d2svg.Render(diagram, &d2svg.RenderOpts{
```

Comment:
the other ones call Render after recursing into layers etc, then prepend not sure if there is a reason for that though

Reply:
do you mean `rendefPDF`?
If so, this is doing the same as there. First the given diagram, then recursively to layers, scenarios and steps